gravatar-quickeditor/src/main/res/drawable/ic_checkmark.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
💡 We can use the material check if you want. `Icons.Filled.Check`

Reply:
Right, but I would have to modify the `PickerPopupItem` to take this or the `DrawableRes`.  Would that be your preference?

Reply:
> I would have to modify the PickerPopupItem to take this or the DrawableRes

I don't think maintaining the two versions is worth it just for this. We can keep it as it is. Thanks! ;)